packages/atomic/tailwind.config.js

```diff
@@ -84,5 +83,6 @@ module.exports = {
         variants('outlineColor')
       );
     }),
+    require('@tailwindcss/line-clamp'),
```

Comment:
we could replace the line-clamp mixin in polyfills.pcss with this

Reply:
We definitely should replace it, and not have two different manner to support line clamp ;)

Reply:
The line clamp mixin is used where there's no Tailwind @btaillon right? The result list